jekyll/_docs/configuration.md

```diff
@@ -191,8 +191,8 @@ hyphens (-) and dots (.).
 CircleCI uses [RVM](https://rvm.io/) to manage Ruby versions.
 We use the Ruby version you specify in your `.rvmrc`, your
 `.ruby-version` file, or your Gemfile.
-If you don't have one of these files, we'll use Ruby `{{ site.data.versions.default_ruby }}`
-or `{{ site.data.versions.old_ruby }}`, whichever we think is better.
+If you don't have one of these files, we'll use Ruby `{{ site.data.precise.versions.default_ruby }}`
+or `{{ site.data.precise.versions.old_ruby }}`, whichever we think is better.
```

Comment:
This maps out to 1.8.7-p358.  Do we still use that?

Also, this refers just to Precise.  Should this section also mention Trusty?